### PR TITLE
done notification controller

### PR DESCRIPTION
**Pull request by chinexman**
### What this PR does (required):
- list PR functionality here

### Screenshots / Videos (front-end only):
- Post at least one screenshot to show the feature in action. **Only required in front-end work**

### Any information needed to test this feature (required):
- Post what steps to take to be able to test this feature (eg. "sign up as a new user and upload a profile picture from the navbar")

### Any issues with the current functionality (optional):
- List any problems where you were not able to complete all tasks on ticket
- List any problems this PR may cause for the project or other active PRs
- Post a screenshot/video of the problem, along with detailed console outputs where applicable

### CODE REVIEW

File: server/controllers/notification.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const Notification = require("../models/Notification");
+const User = require("../models/User");
+const asyncHandler = require("express-async-handler");
+
+exports.notifications = asyncHandler(async (req, res, next) => {
+  const notification = await Notification.findById({ userId: req.user.id });
```
Comment (ugur-akin) [inline server/controllers/notification.js:6]: `findById` will return only a single notification, but users will have more than one notification in most cases.

File: server/routes/notification.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const express = require("express");
+const router = express.Router();
+const protect = require("../middleware/auth");
+const { notifications } = require("../controllers/notification");
+
+router.route("/notification").get(protect, notifications);
```
Comment (ugur-akin) [inline server/routes/notification.js:6]: This is going in the right direction! We can drop the `notification` keyword for this route[^*], `route("/").get` will be enough to convey what this is doing by convention.

[^*]: Usually, express APIs will declare the routes for each resource in `app.js` (e.g. [check out these lines](https://github.com/hatchways/team-reindeer/blob/e28f6518c23640ee2b5743570b4a31ad19dbf156/server/app.js#L47-L49)). All you have to do is import this router in `app.js` and "use" it 👍🏼

**Review by ugur-akin — COMMENTED** [review]
Good start! A couple of quick tips to set you on the right track for the rest of the controllers coming in, good luck 🎉

**Comment by rajivtitus** [comment]
Hey Chinedu! As discussed in our meeting yesterday, please add the additional controllers 😄

**Comment by chinexman** [comment]
ok thanks

**Comment by rajivtitus** [comment]
@sholabioku I'm closing this PR but you can use it as a reference to see how to design the routes/controllers 👍🏽

**Pull request closed without merging**